modules/01-events.js

```diff
@@ -186,19 +188,29 @@ const Module = new Augur.Module()
     }
   } catch (error) { u.errorHandler(error, `User Update Error: ${u.escapeText(newUser?.username)} (${newUser.id})`); }
 })
+.setInit(async () => {
+  const doc = new GoogleSpreadsheet(config.google.sheets.config);
+  try {
+    await doc.useServiceAccountAuth(config.google.creds);
+    await doc.loadInfo();
+    const channels = await doc.sheetsByTitle["Sponsor Channels"].getRows();
+    emojis = Array.from(channels.map(x => [x["Sponsor ID"], x["Sponsor Emoji"]]));
+    emojis = emojis.concat([
+      ["buttermelon", "305039588014161921"],
+      ["noice", "633500960483704843"],
```

Comment:
Let's move these snowflakes into `config/snowflakes-example.json` as `emoji.buttermelon` and `emoji.noice`.